modules/primer-navigation/README.md

```diff
@@ -93,12 +93,123 @@ You can also add optional headings to a menu. Feel free to use nearly any semant
 </nav>
 ```
 
+## Underline nav
+
+`.UnderlineNav` is navigation that is typically used at the top of a page as the main page navigation.
+
+```html title="UnderlineNav"
+<nav class="UnderlineNav">
+  <div class="UnderlineNav-items">
+    <a href="#url" role="tab" title="Item 1" class="UnderlineNav-item selected">Item 1</a>
+    <a href="#url" role="tab" title="Item 2" class="UnderlineNav-item">Item 2</a>
+    <a href="#url" role="tab" title="Item 3" class="UnderlineNav-item">Item 3</a>
+    <a href="#url" role="tab" title="Item 4" class="UnderlineNav-item">Item 4</a>
+  </div>
+</nav>
+```
+
+Use `.UnderlineNav-actions` to use another element alongside the underline nav
+
+```html title="UnderlineNav-actions"
+<nav class="UnderlineNav" aria-label="Foo bar">
+  <div class="UnderlineNav-items">
+    <a href="#url" class="UnderlineNav-item selected">Item 1</a>
+    <a href="#url" class="UnderlineNav-item">Item 2</a>
+    <a href="#url" class="UnderlineNav-item">Item 3</a>
+    <a href="#url" class="UnderlineNav-item">Item 4</a>
+  </div>
+  <div class="UnderlineNav-actions">
+    <a class="btn">Button</a>
+  </div>
+</nav>
+```
+
+You can right align the navigation with `.UnderlineNav--right`.
+
+```html title="UnderlineNav--right"
+<nav class="UnderlineNav UnderlineNav--right">
+  <div class="UnderlineNav-items">
+    <a href="#url" role="tab" title="Item 1" class="UnderlineNav-item selected">Item 1</a>
+    <a href="#url" role="tab" title="Item 2" class="UnderlineNav-item">Item 2</a>
+    <a href="#url" role="tab" title="Item 3" class="UnderlineNav-item">Item 3</a>
+    <a href="#url" role="tab" title="Item 4" class="UnderlineNav-item">Item 4</a>
+  </div>
+</nav>
+```
+
+This also works when adding actions.
+
+```html title="UnderlineNav--right with actions"
+<nav class="UnderlineNav UnderlineNav--right" aria-label="Foo bar">
+  <div class="UnderlineNav-actions">
+    <a class="btn">Button</a>
+  </div>
+  <div class="UnderlineNav-items">
+    <a href="#url" class="UnderlineNav-item selected">Item 1</a>
+    <a href="#url" class="UnderlineNav-item">Item 2</a>
+    <a href="#url" class="UnderlineNav-item">Item 3</a>
+    <a href="#url" class="UnderlineNav-item">Item 4</a>
+  </div>
+</nav>
+```
+
+
+The navigation will work with added counters and/or octicons
+
+```html title="UnderlineNav with Counter"
+<nav class="UnderlineNav" aria-label="Foo bar">
+  <div class="UnderlineNav-items">
+    <a href="#url" class="UnderlineNav-item selected">
+      <%= octicon "tools" %>
+      Item 1
+    </a>
+    <a href="#url" class="UnderlineNav-item">
+      <%= octicon "tools" %>
+      Item 2
+      <span class="Counter">10</span>
+     </a>
+     <a href="#url" class="UnderlineNav-item">
+       <%= octicon "tools" %>
+       Item 3
+    </a>
+    <a href="#url" class="UnderlineNav-item">
+      <%= octicon "tools" %>
+      Item 4
+     </a>
+  </div>
+</nav>
+```
+
+Use `.UnderlineNav--full` to use a container within the components
```

Comment:
Suggesting change wording to something like:

> Use `UnderlineNav--full` in combination with container styles to make navigation fill the width of the container.

I couldn't find a way to avoid using "container" twice but I think it's okay.